src/actions/CryptoExchangeActions.tsx

```diff
@@ -70,7 +70,7 @@ export function getQuoteForTransaction(navigation: NavigationBase, info: SetNati
       })
       dispatch({ type: 'UPDATE_SWAP_QUOTE', data: swapInfo })
     } catch (error: any) {
-      navigation.navigate('exchange', {})
+      navigation.navigate('exchangeTab', {})
```

Comment:
I am not sure what happens if you navigate to a tab that you are already on. Does it always always always reset the state and go to the first scene? If not, you can force it to do so by passing `{ screen: 'xyz' }` as the route props.

If it does reset to the first scene every time, how does `MenuTabs` avoid resetting the state? Would it make sense to use the `.jump` action there to preserve the state?

Reply:
I forgot to add the feature to navigate back to top on the second tap.